deployment/helm/balloons/templates/daemonset.yaml

```diff
@@ -67,11 +73,14 @@ spec:
             - --config-group-label
             - {{ .Values.configGroupLabel }}
             {{- end }}
+        {{- if (not (eq .Values.ports nil)) }}
           ports:
-            - containerPort: 8891
-              protocol: TCP
-              hostPort: {{ .Values.hostPort }}
-              name: metrics
+          {{- range $port := .Values.ports }}
+            - name: {{ $port.name }}
+              container: {{ $port.container }}
```

Comment:
```suggestion
              containerPort: {{ $port.containerPort }}
```

Reply:
Ah, thanks for spotting. Fixed (but I still chose to not tautologically repeat 'Port' in the Helm value schema port property names).